prow/plugins/retitle/retitle.go

```diff
@@ -135,13 +135,12 @@ func handleGenericComment(gc githubClient, isTrusted func(string) (bool, error),
 		pr.Title = newTitle
 		_, err = gc.EditPullRequest(org, repo, number, pr)
 		return err
-	} else {
-		issue, err := gc.GetIssue(org, repo, number)
-		if err != nil {
-			return err
-		}
-		issue.Title = newTitle
-		_, err = gc.EditIssue(org, repo, number, issue)
+	}
+	issue, err := gc.GetIssue(org, repo, number)
+	if err != nil {
 		return err
```

Comment:
fyi: in general I think it is better form when a function can return a variety of errors to help disambiguate between them, aka: return `fmt.Errorf("get issue: %v", err")` vs `fmt.Errorf("edit issue: %v", err)` later

But nothing in this file does that yet so makes sense to follow existing convention (but would make a good follow up PR if you're interested)